bin/ensure-node-modules-present.js

```diff
@@ -0,0 +1,13 @@
+const spawnSync = require( 'child_process' ).spawnSync;
+const fs = require( 'fs' );
+
+if ( ! fs.existsSync( 'node_modules' ) ) {
```

Comment:
What are your thoughts regarding using `install-if-needed` in npm commands, but doing it differently here? Would it make sense to put this logic (and make it more explicit) in the `package.json` itself? That way we'd only need to look at one place to know what and when packages are installed.

Kind of doing:

- install-if-shrinkwrap-differs && npm command
- install-if-no-packages && npm command